lib/data/stats.ts

```
@@ -1,9 +1,11 @@
-import { and, count, eq, sql } from "drizzle-orm";
+import { and, count, eq, isNull, sql } from "drizzle-orm";
 import { db } from "@/db/client";
-import { conversations, conversationMessages as emails } from "@/db/schema";
+import { conversations, conversationMessages as emails, userProfiles } from "@/db/schema";
 import { getFullName } from "@/lib/auth/authUtils";
 import { Mailbox } from "@/lib/data/mailbox";
 import { UserRole, UserRoles } from "@/lib/data/user";
+import { authUsers } from "@/db/supabaseSchema/auth";
+import { access } from "fs";
```

ESLint

[error] Delete `";⏎import·{·authUsers·}·from·"@/db/supabaseSchema/auth";⏎import·{·access·}·from·"fs`

(prettier/prettier)